### PR TITLE
ENT-1009 Grab enterprise context/logo based on slug in url

### PR DESCRIPTION
**Pull request by brittneyexline**

### CODE REVIEW

**Comment by brittneyexline** [comment]
@georgebabey @adamstankiewicz - i have successfully got this working locally, but have not had a chance to fix/write tests. feel free to pick this up given I will be out for next week.

File: src/containers/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const Header = (props) => {
+  const { enterpriseLogo, enterpriseSlug } = props;
+
+  return (
+    <header className="container">
+      <nav className="navbar px-0 justify-content-between">
+        <Link
+          to={enterpriseSlug ? `/${enterpriseSlug}` : '/'}
+          replace
```
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply src/containers/Header/index.jsx]: The `replace` prop makes the link work more like a traditional anchor tag in that when you click on the link repeatedly when you're already on the same route, it doesn't add every click to the history. 

I ran into the case where clicking on the route several times from the same page made it so I had to click the back button multiple times to get back to the real previous page

https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md#replace-bool

[the remark this answers is not in the document]
Comment (georgebabey) [reply src/containers/Header/index.jsx]: (Capturing what we discussed so theres a paper trail)

The problem with replace here is if I'm on a subpage and click the header logo my browser url changes but that previous page isn't pushed into history so the back button won't return me to where I just was.

File: src/components/Admin/index.jsx [hunk 2]
```diff
@@ -21,10 +21,18 @@ const StatusMessage = props => (
...
-    const enterpriseId = 'ee5e6b3a-069a-4947-bb8d-d2dbc323396c';
-    this.props.getDashboardAnalytics(enterpriseId);
+    const { enterpriseId } = this.props;
+
+    if (enterpriseId) {
+      this.props.getDashboardAnalytics(enterpriseId);
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
```
[the remark this answers is not in the document]
Comment (georgebabey) [reply src/components/Admin/index.jsx]: Agreed, but I think its especially important now that this method is making a fetch call. As-is we could end up making multiple unnecessary calls to `getDashboardAnalytics()`

**Pull request merged**